src/pipx/main.py

```diff
@@ -415,9 +416,19 @@ def _add_inject(subparsers, venv_completer: VenvCompleter, shared_parser: argpar
     ).completer = venv_completer
     p.add_argument(
         "dependencies",
-        nargs="+",
+        nargs="*",
         help="the packages to inject into the Virtual Environment--either package name or pip package spec",
     )
+    p.add_argument(
+        "-r",
+        "--requirement",
+        dest="requirements",
+        action="append",
+        default=[],
+        metavar="file",
+        help="file containing the packages to inject into the Virtual Environment--"
+        "one package name or pip package spec per line.",
+    )
```

Comment:
Please add an example on how to use this and how a inject-requirements file would look like to our docs as well. Probably to `docs/examples.md`.